tests/unit/modules/dcnm/fixtures/dcnm_inventory.json

```diff
@@ -7,7 +7,7 @@
         "preserve_config": false,
         "role": "leaf",
         "seed_ip": "192.168.1.110",
-        "user_name": "admin"
+        "user_name": "username"
```

Comment:
Do the unit tests still pass with these changes in the fixtures file?

Reply:
yes mike, they pass.

(ansible_work) (base) KHARICHA-M-N1VA:dcnm kharicha$ pytest tests/unit/modules/dcnm/test_dcnm_inventory.py 
=============================================================================== test session starts ================================================================================
platform darwin -- Python 3.8.0, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
rootdir: /Users/kharicha/Desktop/dcnm_phase2_servicenode/collections/ansible_collections/cisco/dcnm
plugins: xdist-1.31.0, forked-1.1.3
collected 27 items                                                                                                                                                                 

tests/unit/modules/dcnm/test_dcnm_inventory.py ...........................                                                                                                   [100%]

================================================================================ 27 passed in 0.32s ================================================================================
(ansible_work) (base) KHARICHA-M-N1VA:dcnm kharicha$